src/style.css

```
@@ -0,0 +1,100 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;
+  font-style: normal;
+  --line_heigth_header: 60px;
+}
+
+.header {
+  display: flex;
+  flex-wrap: wrap;
+  left: 0;
+  right: 0;
+  padding-right: 50px;
+  padding-left: 50px;
+  position: fixed;
```

```suggestion
  padding-right: 50px;
  padding-left: 50px;
```
for this task header may be without positioning, but ok